pages/api/create-account.js

```diff
@@ -0,0 +1,22 @@
+import { Auth } from 'aws-amplify'
+
+export default async function handler(req, res) {
+  if (req.method !== 'POST') return res.status(405).send();
+
+  const { username, password, lastName, firstName } = JSON.parse(req.body);
```

Comment:
Beautiful

nit: Have some form of consistency when listing variables. Either alphabetical or by use-case. It makes it easier across the codebase to look for certain variables or add new ones in the correct location.

Alphabetical: `firstName, lastName, password, username`
Use-case: `username, password, firstName, lastName`

Reply:
Changed to Alphabetical for consistency